docs/registration.adoc

```diff
@@ -0,0 +1,125 @@
+
+// TODO: Add instruction
+
+[#register-operator]
+== Register Operator
+
+Once the tokens are staked the one controlling the Staking Provider account has to
+register an operator for the Random Beacon and TBTC applications.
+
+If a Key File for the Staking Provider account is available the registration
+can be performed from the <<register-operator-client>>.
+
+Another way is registration from the <<register-operator-web>>.
+
+NOTE: For the Keep Legacy stakes the Staking Provider account and Operator account are
+the same.
+
+[#register-operator-client]
+=== Keep Client
+
+An operator registering transaction can be submitted with the Keep Client.
+
+The commands that should be used are provided below.
+You need to replace few values in the commands:
+
+* `STAKING_PROVIDER_KEY_FILE_ACCOUNT_PASSWORD` - a password used for the Staking Provider
+Ethereum Key File encryption,
+
+* `STAKING_PROVIDER_KEY_FILE_PATH` - a path to the Staking Provider Ethereum
+Key File,
+
+* `ETHEREUM_API_WS_URL` - an URL to the Ethereum WebSocket API,
+
+* `OPERATOR_ADDRESS` - an Operator address that will be running the Keep Client.
+
+
+Command templates for the supported applications:
+
+* Random Beacon application (`RandomBeacon` contract)
+
+[source,bash]
+----
+export KEEP_CLIENT_CONFIG_DIR=$(pwd)/config
+
+export KEEP_CLIENT_ETHEREUM_WS_URL="<Ethereum API WS URL>"
+
+export KEEP_CLIENT_ETHEREUM_PASSWORD="<Staking Provider Account Key File Password>"
+export STAKING_PROVIDER_KEY_FILE_NAME="<Staking Provider Account Key File Name>"
+export OPERATOR_ADDRESS="<Operator Account Address>"
+
+docker run \
+    --volume $KEEP_CLIENT_CONFIG_DIR:/mnt/keep-client/config \
+    us-docker.pkg.dev/keep-test-f3e0/public/keep-client:latest \
+    ethereum \
+    --ethereum.url $KEEP_CLIENT_ETHEREUM_WS_URL \
+    --ethereum.keyFile /mnt/keep-client/config/$STAKING_PROVIDER_KEY_FILE_NAME \
+    beacon random-beacon register-operator --submit \
+    $OPERATOR_ADDRESS
+----
+
+
+* TBTC application (`WalletRegistry` contract)
+
+[source,bash]
+----
+export KEEP_CLIENT_CONFIG_DIR=$(pwd)/config
+
+export KEEP_CLIENT_ETHEREUM_WS_URL="<Ethereum API WS URL>"
+
+export KEEP_CLIENT_ETHEREUM_PASSWORD="<Staking Provider Account Key File Password>"
+export STAKING_PROVIDER_KEY_FILE_NAME="<Staking Provider Account Key File Name>"
+export OPERATOR_ADDRESS="<Operator Account Address>"
+
+docker run \
+    --volume $KEEP_CLIENT_CONFIG_DIR:/mnt/keep-client/config \
+    us-docker.pkg.dev/keep-test-f3e0/public/keep-client:latest \
+    ethereum \
+    --ethereum.url $KEEP_CLIENT_ETHEREUM_WS_URL \
+    --ethereum.keyFile /mnt/keep-client/config/$STAKING_PROVIDER_KEY_FILE_NAME \
+    ecdsa wallet-registry register-operator --submit
+    $OPERATOR_ADDRESS
+----
+
+NOTE: icon:flask[] For Testnet add `--goerli` flag to the commands executed above.
+
+[#register-operator-web]
+=== Web Browser
+
+An operator registering transactions can be submitted with Etherscan.
```

Comment:
Original sentence is a bit ambiguous (operator that registers transactions vs transaction that registers operator). For better readability I would use hyphen.
```suggestion
An operator-registering transactions can be submitted with Etherscan.
```